bliss.js

```diff
@@ -483,57 +507,43 @@ $.setProps = {
 		}
 	},
 
-	once: function(val) {
-		if (arguments.length == 2) {
-			val = {};
-			val[arguments[0]] = arguments[1];
-		}
-
+	once: overload(function(events, callback){
```

Comment:
After looking at ```$.events``` for a while, Im not really understanding how that method works / what it actually does. And the tests are even more confusing. Im gonna need some help / explanation on this one.